apps/web/components/ServiceInformation/ServiceInformation.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import {
+  Box,
+  GridColumn,
+  GridRow,
+  Inline,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import { ApiService, GetNamespaceQuery } from '@island.is/web/graphql/schema'
+import { useNamespace } from '../../hooks'
+
+const capitalize = (s: string) => {
+  return s.charAt(0).toUpperCase() + s.slice(1).toLowerCase()
+}
+
+export interface ServiceInformationProps {
+  service: ApiService
+  strings: GetNamespaceQuery['getNamespace']
+}
+
+export const ServiceInformation = ({
+  service,
+  strings,
+}: ServiceInformationProps) => {
+  const n = useNamespace(strings)
+
+  const showTags = () => {
```

Comment:
I would like to see this inline inside the return, defined outside the component, as a separate component or inside useCallback. In this case this function will be redefined every time the component is rendered. It may not cause a major performance hit but it can lead to undesirable side effects

Reply:
Personally I don't see the benefit in making this a function